src/Accounts/Accounts/CommonModule/UserAgent.cs

```diff
@@ -43,8 +53,13 @@ public class UniqueId
         /// <returns>Amended pipeline for retrieving a response</returns>
         public Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken token, Action cancel, SignalDelegate signal, NextDelegate next)
         {
-            // add a header...
-            request.Headers.Add("x-ms-unique-id", Interlocked.Increment(ref this.count).ToString());
+            var userAgents = new ProductInfoHeaderValue[] { new ProductInfoHeaderValue("AzurePowerShell", $"Az4.0.0-preview") };
```

Comment:
@markcowl, do we expect to get "Az4.0.0-preview" userAgent in coming Az 2.x/3.x release?

Reply:
No, this is a separate pipeline, not used by existing cmdlets - only generated cmdlets use this pipeline